consensus/src/marshal/ingress/mailbox.rs

```diff
@@ -22,6 +22,20 @@ pub enum Identifier<D: Digest> {
     Latest,
 }
 
+// Allows using u64 directly for convenience.
+impl<D: Digest> From<u64> for Identifier<D> {
+    fn from(src: u64) -> Self {
+        Self::Height(src)
+    }
+}
+
+// Allows using &Digest directly for convenience.
+impl<D: Digest> From<&D> for Identifier<D> {
```

Comment:
must do `From<&D>` instead of `From<D>` since `u64` is a valid `Digest` and we already implement `From<u64>` above